pgmpy/estimators/MLE.py

```diff
@@ -7,35 +9,35 @@
 
 class MaximumLikelihoodEstimator(BaseEstimator):
     """
-    Class used to compute parameters for a model using Maximum Likelihood Estimate.
+    Class used to compute parameters for a model using Maximum Likelihood Estimation.
 
     Parameters
     ----------
     model: A pgmpy.models.BayesianModel instance
 
     data: pandas DataFrame object
-        datafame object with column names same as the variable names of the network
+        DataFrame object with column names identical to the variable names of the network
 
     Examples
     --------
     >>> import numpy as np
     >>> import pandas as pd
     >>> from pgmpy.models import BayesianModel
     >>> from pgmpy.estimators import MaximumLikelihoodEstimator
-    >>> values = pd.DataFrame(np.random.randint(low=0, high=2, size=(1000, 5)),
+    >>> data = pd.DataFrame(np.random.randint(low=0, high=2, size=(1000, 5)),
     ...                       columns=['A', 'B', 'C', 'D', 'E'])
     >>> model = BayesianModel([('A', 'B'), ('C', 'B'), ('C', 'D'), ('B', 'E')])
-    >>> estimator = MaximumLikelihoodEstimator(model, values)
+    >>> estimator = MaximumLikelihoodEstimator(model, data)
     """
-    def __init__(self, model, data):
+    def __init__(self, model, data, node_values=None):
```

Comment:
@chrisittner Is `node_values` is for passing the state names ? If yes, I would suggest you to change the name as it's not clear from the name.